### PR TITLE
Add release notes for stable 1.2.0

### PR DESCRIPTION
**Pull request by joejulian**
**What type of PR is this?**

Documentation

**What this PR does/ why we need it**:

Release notes

**Which issue(s) this PR fixes**:

no issue

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

* [x] The commit message explains the changes and why are needed.
* [x] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [x] The documentation is updated where needed.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## stable-1.15-1.2.0, stable-1.16-1.2.0
+
+* fluent-bit
+  * Disabled audit log  collection
+    It's been observed in production clusters that the audit log bloats the number of fields in an index.
+    This causes resource limits to be filled and throttling to occur.
+    We are disabling this collection pending further investigation.
+* dex:
+  * improves the LDAP connector validation in Dex controller
+  * Fixes an issue in dex addon which disallowed adding local users
```
Comment (dkoshkin) [inline RELEASE_NOTES.md]:
```suggestion
  * fix an issue in dex addon which disallowed adding local users
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Release Notes
+## stable-1.15-1.2.0, stable-1.16-1.2.0
+
+* fluent-bit
+  * Disabled audit log  collection
+    It's been observed in production clusters that the audit log bloats the number of fields in an index.
+    This causes resource limits to be filled and throttling to occur.
+    We are disabling this collection pending further investigation.
+* dex:
+  * improves the LDAP connector validation in Dex controller
```
Comment (dkoshkin) [inline RELEASE_NOTES.md]:
```suggestion
  * improve the LDAP connector validation in Dex controller
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Release Notes
+## stable-1.15-1.2.0, stable-1.16-1.2.0
+
+* fluent-bit
+  * Disabled audit log  collection
```
Comment (dkoshkin) [inline RELEASE_NOTES.md]:
```suggestion
  * Disable audit log  collection
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  * improves the LDAP connector validation in Dex controller
+  * Fixes an issue in dex addon which disallowed adding local users
+  * use Dex controller v0.4.1, which includes the support for OIDC group claims
+  * upgrade Dex to v2.22.0, which supports groups claims for OIDC connectors
+* dex-k8s-authenticator: 
+  * allow scopes to be configured, and drop the `offline_access` scope as it is not used
+* kube-oidc-proxy:
+  *  enable token passthrough
+* opsportal:
+  * sets opsportalRBAC.allowAllAuthenticated to true
```
Comment (dkoshkin) [inline RELEASE_NOTES.md]:
```suggestion
  * set `opsportalRBAC.allowAllAuthenticated` to true
```

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+* traefik-forward-auth:
+  * enable RBAC and impersonation
+  * remove whitelisting
+* kibana:
+  * upgrade to 6.8.2
+* elasticsearch-curator:
+  * added and enabled curator to remove old indexes from elasticsearch to free up storage
+
+
+Added support for kubernetes clusters on GCP
```
Comment (dkoshkin) [inline RELEASE_NOTES.md]:
```suggestion
Add support for kubernetes clusters on GCP
```

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Release Notes
+## stable-1.15-1.2.0, stable-1.16-1.2.0
+
+* fluent-bit
+  * Disable audit log  collection
+    It's been observed in production clusters that the audit log bloats the number of fields in an index.
+    This causes resource limits to be filled and throttling to occur.
+    We are disabling this collection pending further investigation.
```
Comment (samvantran) [inline RELEASE_NOTES.md]: Just a note, users can still turn this on if they wish to. We are just turning it off by default.

**Pull request merged**